examples/tests/requirements.txt

```diff
@@ -4,8 +4,7 @@ protobuf<3.20; sys_platform == 'darwin' and platform_machine == 'arm64'
 pytest>=6.0.1
 tensorflow==2.8.4; sys_platform != 'darwin' or platform_machine != 'arm64'
 tensorflow-macos==2.8.0; sys_platform == 'darwin' and platform_machine == 'arm64'
-torch==1.7.1
```

Comment:
Is the goal to test our examples against the latest torch at all times?

That seems a lot riskier than unpinning pandas...

Reply:
I see the original review comment...

Where is this file actually pulled in?  That would shed some light here.

Reply:
It looks to me from our `.circleic/config.yml` that we actually do not include torch anywhere else during the `test-examples` job, and the torch being installed is just the latest `torch==1.13.1`, which is being pulled in implicitly by the `torchvision` requirement in this file.

Is that your intention here?

Reply:
That is absolutely not my intention, and I don't understand how that happened. My intention was for it to use whatever version of torch is pinned in the root `requirements.txt`, which I'd thought would be what was installed on the image when `pip install. determined` was run.

Thanks for catching it. Investigating.

Reply:
Ah -- I see that head `torchvision` has a requirement on `torch==2.0.0`, which would explain why the version of torch could be overwritten with a `pip install` after having installed `determined`, depending on what pip chooses to do when there are irreconcilable version requirements.

I'm not totally sure what the best choice is, here. Probably going to re-pin torchvision to something consistent with `torch==1.9.0`, but from there whether or not to re-include an explicit `torch==1.9.0` in the requirements here or instead drop a comment in this particular `requirements.txt`, I dunno.

... probably re-pin `torch==1.9.0` and just let the requirement be repeated?

Reply:
More clarity. Per https://github.com/determined-ai/determined/pull/3811, the torch requirement was put directly into the Makefile rather than into requirements.txt for reasons I don't really understand. Not going to second-guess those reasons now, but as a result of torch not being pinned in determined's `requirements.txt` we can't depend on it already being installed when installing these requirements. I think duplication does need to occur, and I'll drop a note into the Makefile explaining the dupe.

Reply:
Also, per a reminder from Ryan OOB, requirements.txt is relevant for building a module, not installing it with pip. Those dependencies are in setup.py, which in this case does not include torch at all.